src/reactviews/pages/ExecutionPlan/iconMenu.tsx

```diff
@@ -0,0 +1,171 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useState } from 'react';
+import { ExecutionPlanContext } from "./executionPlanStateProvider";
+import * as utils from './queryPlanSetup';
+import './executionPlan.css';
+import { makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+	iconStack: {
+		right: 0,
+		display: "flex",
+		flexDirection: "column",
+		alignItems: "center",
+		width: "25px",
+		opacity: 1,
+		zIndex: "9999",

```

Comment:
how did we arrive at this value for zIndex?  i.e., was it recommend by vscode extensibility documentation as a way to always be on top of all their UI or something?

Reply:
I changed it to a more reasonable value.